app/models/form.rb

```diff
@@ -44,20 +42,15 @@ def append_page(page)
   end
 
   def ready_for_live?
-    @missing_sections = []
+    get_task_status_service.mandatory_tasks_completed?
```

Comment:
The idea behind having these methods in the form model is that once forms-api does the validation, these methods will be replaced by equivalent data on the form object itself.